File: MultiFlex/API/JavaRestApi/src/main/java/org/acme/DTO/ShelfDto.java
package org.acme.DTO;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.acme.model.Ware;

import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ShelfDto implements Serializable {

    private Integer id;
    private Integer position;
    private Integer maxAmount;

    //private RegalDto regal;
    //private WareDto ware;

    private Integer ware_id;
    private Integer regal_id;
}

File: MultiFlex/Frontend/Multiflex.Frontend/MultiFlex.Frontend.WebApp/Models/Dto/ShelfDto.cs
namespace Multiflex.Frontend.WebApp.Models.Dto
{
    public class ShelfDto : IdentityModel
    {
        public int maxAmount { get; set; }
        public int position { get; set; }
        public int regal_id { get; set; }
        public int ware_id { get; set; }
    }
}
